Gemfile

```diff
@@ -7,12 +7,12 @@ gem 'addressable', '~> 2.3'
 gem 'coder'
 gem 'jemalloc'
 gem 'metriks', '0.9.9.6'
-gem 'metriks-librato_metrics', github: 'eric/metriks-librato_metrics'
+gem 'metriks-librato_metrics', git: 'https://github.com/eric/metriks-librato_metrics'
```

Comment:
No, the git protocol is unauthenticated. It does not use SSH, and it does not use TLS. Here is a wireshark capture showing the plaintext information on the wire, from running `git clone git://github.com/travis-ci/travis-support.git`:

![screen shot 2016-06-30 at 18 18 02](https://cloud.githubusercontent.com/assets/11158255/16495826/51912636-3eef-11e6-9b07-2723db96944f.png)